domains/fajox.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "FajoX1",
+        "email": "pybytee@gmail.com"
+    },
+    "record": {
+        "A": ["185.199.108.153"]
```

Comment:
I cannot reach your site (404 page)

Reply:
it's github's "A" dns. fajox1.github.io

Reply:
> it's github's "A" dns. fajox1.github.io. I will add it when will receive domain.

github pages uses 4 IPs for the A record, `185.199.108.153`, `185.199.109.153`, `185.199.110.153`, `185.199.111.153`

Reply:
> > it's github's "A" dns. fajox1.github.io. I will add it when will receive domain.
> 
> github pages uses 4 IPs for the A record, `185.199.108.153`, `185.199.109.153`, `185.199.110.153`, `185.199.111.153`

yes, can you give me domain and i will change dns to github's dns

Reply:
@FajoX1 You need to fix your domain first.

Reply:
> I cannot reach your site (404 page)

Now site work (fajox1.github.io) and i wrote dns, so, can you give me domain fajox.is-a.dev?

Reply:
@wdhdev

Reply:
you have to wait for a maintainer to merge it now